includes/class-wc-stripe-webhook-handler.php

```
@@ -1437,17 +1437,26 @@ public function process_account_updated( $notification ) {
 	 * sessions, as well as agentic checkout sessions.
 	 *
 	 * @param object $notification The notification from Stripe
 	 * @return void
 	 */
-	public function process_checkout_session( object $notification ): void {
+	public function process_checkout_session_success( object $notification ): void {
 		$checkout_session = $notification->data->object;
-		$session_id       = $checkout_session->id;
+
+		if ( ! isset( $checkout_session->id ) ) {
+			WC_Stripe_Logger::error( 'Checkout session ID is missing from the event data.', [ 'notification' => $notification ] );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'process_checkout_session_success'. (undefined)

(StaticAccess)